Resources/Patches/save/credits/strings.event

```diff
@@ -112,7 +112,7 @@ defeatedCreditsEnglish:
 String("Defeated"); BYTE 0
 
 defeatedCreditsFrench:
-String(""); BYTE 0
+String("Vaincu"); BYTE 0
```

Comment:
Défait: (kind of a weird one but you have to find a synonym

Reply:
ok modifié